src/internal/carriers/packages/package-item.ts

```diff
@@ -25,20 +25,23 @@ export class PackageItem implements IPackageItem {
       salesOrder: SalesOrderIdentifier[_internal].schema.unknown(true),
       salesOrderItem: SalesOrderItemIdentifier[_internal].schema.unknown(true),
       product: ProductIdentifier[_internal].schema.unknown(true),
-      quantity: Quantity[_internal].schema.required(),
-      unitPrice: MonetaryValue[_internal].schema.required(),
+      quantity: Quantity[_internal].schema.optional(),
+      unitPrice: MonetaryValue[_internal].schema.optional(),
     }),
   };
 
-  public readonly sku: string;
-  public readonly identifiers: Identifiers;
+  public readonly sku?: string;
+  public readonly identifiers?: Identifiers;
```

Comment:
@justin-shipstation It's not necessary to make these optional. `sku` will be initialized to an empty string, `identifers` will just be an empty object.